src/ml_tooling/transformers/rare_feature.py

```diff
@@ -0,0 +1,37 @@
+import pandas as pd
+from sklearn.base import BaseEstimator, TransformerMixin
+
+
+class RareFeatureEncoder(BaseEstimator, TransformerMixin):
+    """
+    Replace values which occurs below the chosen threshold.
```

Comment:
Try to write the docstring more like a description instead of bullets - see some of the other transformers for inspiration

Reply:
Instead?: 
Replaces categories with a value, if they occur less than a threshold.